core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -1123,6 +1152,19 @@ protected function doReplace($strBuffer, $blnCache)
 
 		return StringUtil::restoreBasicEntities($strBuffer);
 	}
+
+	/**
+	 * @return array<string|null, array>
+	 */
+	private function parseUrlWithQueryString(string $url): array
+	{
+		$base = parse_url($url, PHP_URL_PATH) ?: null;
+		$query = parse_url($url, PHP_URL_QUERY) ?: '';
+
+		parse_str($query, $attributes);
+
+		return array($base, $attributes);
```

Comment:
If I compare this with the code of the `image` tag, I notice that there are some differences.

https://github.com/contao/contao/blob/6d3ccc6a115719c0f1a5a4e0be90ba7928750a56/core-bundle/src/Resources/contao/library/Contao/InsertTags.php#L783-L786

What about decoding and replacing `[&]` with `&`? Is this not necessary here?